### PR TITLE
l10n string updates

### PR DESCRIPTION
**Pull request by keianhzo**
l10n string updates. Related to https://github.com/mozilla-l10n/android-l10n/pull/192

### CODE REVIEW

**Comment by keianhzo** [comment]
@flodolo please help review.

File: app/src/main/res/values-da/strings.xml [hunk 1]
```diff
@@ -108,7 +108,7 @@
 
     <!-- This string is used as the title of the pop-ups dialog used to remove sites from the currently
          allowed pop-up sites. -->
-    <string name="settings_privacy_policy_popups_title">Pop op-tilladelser</string>
+    <string name="settings_privacy_policy_popups_title_v1">Pop op-tilladelser</string>
```
Comment (flodolo) [inline app/src/main/res/values-da/strings.xml:111]: As a general rule, you shouldn't modify localized files. But this is definitely wrong:
- You're adding a new string in en-US, `settings_privacy_policy_popups_title_v1` with text `Exceptions for pop-ups`. The old one was `settings_privacy_policy_popups_title`, `Pop-ups permissions`.
- By doing these changes, all locales will have this new string "pre-translated", but with the wrong reference (the old one, `Pop-ups permissions`.

Please revert all changes to the localized files, only add `settings_privacy_policy_popups_title_v1` to the English/source file as you already did.

Comment (keianhzo) [reply app/src/main/res/values-da/strings.xml:111]: Thanks, fixed.

**Comment by keianhzo** [comment]
@flodolo I've updated the PR. I've added an issue to move html encoded entities to CDATA if that plays better with the localization flow: https://github.com/MozillaReality/FirefoxReality/issues/3185

**Comment by keianhzo** [comment]
@bluemarvin This is ready to be landed.

**Pull request merged**